aries-site/src/examples/templates/FilterControls/FilterControls.js

```diff
@@ -62,6 +64,7 @@ export const FilterControls = ({
             <SearchFilter placeholder={searchFilter.placeholder} />
           )}
           <Filters />
+          {configure}
```

Comment:
What do you all think about the control order here? Ian called out how the buttons get spaced when filters are applied: 
<img width="1369" alt="Screen Shot 2022-07-15 at 10 36 17 AM" src="https://user-images.githubusercontent.com/1756948/179269690-a9d2b856-a6d6-4101-8691-fa917288bfd2.png">

I am conflicted, as search & filter are more closely related in function, but the "clear filters" button certainly throws off the visual...

@britt6612 @taysea @bahriaditi @MeganBeierle

Reply:
agree I wonder if we can move the `clear filters` to the last option. Visually this doesn't flow

Reply:
I think the 80% case is not to have the column order/select control. So that's a relief.
For the case where it is there, I tend to think no options are ideal, as filtering is likely to be used way more than column selection and is more associated with searching, so it makes sense for both proximity and frequency to have it next to search. I do think "Clear Filters" should be next to the filter control. If I had to choose, I'd probably go with what we have in the screenshot above instead of swapping the filter and column buttons. But, I that is a weak preference.

Reply:
Oh good catch @halocline -- I think I've seen the "Configure" button appearing between the search + filter in some incoming contributions. I like that approach because then we have a smoother visual flow.

Reply:
I'm OK with configure before filter

Reply:
Good call @bahriaditi 

The example you provided with column sorting on the right looks so much better. Do we know how often customers choose to configure columns? Just curious.

Reply:
> We should probably at some point reassess placement of 'configure' - whether it belongs on the right or the left. I see that there is an [example on our site](https://design-system.hpe.design/templates/filtering?q=filt#filtering-with-selectable-results) which has it on the right.

@bahriaditi that is what this PR is doing is changing how we currently have it. which is not having the selector by the action buttons

Reply:
Well, this is embarrassing 🤷🏼‍♀️ - Friday brain